src/a-runtime-console/kubernetes/ui/pipeline/input-action-dialog/input-action-dialog.component.ts

```diff
@@ -22,6 +25,7 @@ export class InputActionDialog {
   @ViewChild('inputModal') modal: any;
 
   constructor(private http: Http,
+              private notifications: Notifications,
               private authService: AuthenticationService,
               @Inject(FABRIC8_FORGE_API_URL) private forgeApiUrl: string
```

Comment:
I believe `FABRIC8_FORGE_API_URL` needs to be completely removed, since it is no longer used